main.py

```diff
@@ -1,26 +1,37 @@
+import psutil
 import win32api
+import win32con
 import win32process
 import win32gui
 import argparse
 
 
+# pid = None
+
 def build_parser():
     parser = argparse.ArgumentParser(description='This is a title changer')
-    parser.add_argument('win_name', action="store", type=str)
+    parser.add_argument('proc_name', action="store", type=str)
     parser.add_argument('new_title', action="store", type=str)
 
     parse_results = parser.parse_args()
 
     return parse_results
 
 
-def find_it(hwnd, parser):
-    if parser.win_name in win32gui.GetWindowText(hwnd):
-        win32gui.SetWindowText(hwnd, parser.new_title)
+def enum_handler(hwnd, list_of_params):
+    _, p = win32process.GetWindowThreadProcessId(hwnd)
+    if p is not None and p == list_of_params[0]:
+        win32gui.SetWindowText(hwnd, list_of_params[1])
+
 
 def main():
     parser = build_parser()
-    win32gui.EnumWindows(find_it, parser)
+
+    for proc in psutil.process_iter():
```

Comment:
psutil is not a part of pywin32, I would love to see you use this functions in your actual work, but  please use the functions under win32process for this exercise